js/books.js

```diff
@@ -68,7 +60,7 @@ class Interface {
     const header = document.querySelector('.titletag');
     containerELement.insertBefore(newdiv, header);
     // Vanish in 3 seconds
-    setTimeout(() => document.querySelector('.alert').remove(), 3000);
+    setTimeout(() => document.querySelector('.alert').remove(), 2000);
   }
 }
 
```

Comment:
- [ ] [OPTIONAL]
It's a good idea to use non mutating array methods `concat`, over `push`, and similarly `slice` over `splice` on _lines 82 and 91_ respectively, so as to keep the original array intact.